pifi/videoplayer.py

```diff
@@ -82,7 +82,12 @@ def show_loading_screen(self):
         if self.__led_settings.is_color_mode_rgb():
             filename = 'loading_screen_color.npy'
         loading_screen_path = DirectoryUtils().root_dir + '/' + filename
-        self.play_frame(np.load(loading_screen_path))
+        img = np.load(loading_screen_path)
+        height, width = self.__led_settings.display_height, self.__led_settings.display_width
+        frame = np.zeros([height, width, 3], np.uint8)
+        borderY, borderX = (height-img.shape[0]) // 2, (width - img.shape[1]) // 2
+        frame[borderY:-borderY,borderX:-borderX] = img
+        self.play_frame(frame)
```

Comment:
Can you explain why this is necessary? I thought my changes here would be sufficient: https://github.com/dasl-/pifi/commit/9ef5ae1d9b7636d939d8cd2b98b13e3fd772ac80

The idea is that the loading image will be generated, scaled to the dimensions of your LED matrix. On my pi it displays centered, curious how it looks on yours?

Reply:
It displays in the lower right corner on mine

Reply:
I think I found the problem.

1) there was a bug with my recent `img_to_led` commit: https://github.com/dasl-/pifi/commit/9ef5ae1d9b7636d939d8cd2b98b13e3fd772ac80#diff-4b1182546e0672abf15b60223a597c1f27dd073cdfa4e0832809a9491a82c04c

I've fixed that bug now: 23497f2fea2a1f1b48360157ac23c3f526a8a32c

2) you need to `rm loading_screen_color.npy loading_screen_monochrome.npy` to ensure that they get regenerated by the `./install/install.sh` script: https://github.com/dasl-/pifi/blob/23497f2fea2a1f1b48360157ac23c3f526a8a32c/install/install.sh#L60

After deleting those files, rerun the `./install/install.sh` script.

Hopefully it should be centered after all that... Assuming that works, maybe we can close this PR.

Reply:
Yep, this fixes it!